pages/project-version/middleware/index.js

```
@@ -115,17 +124,20 @@ const getPreviousVersion = req => {
   if (!previous) {
     return Promise.resolve();
   }
-
   return req.api(`/establishments/${req.establishmentId}/projects/${req.projectId}/project-versions/${previous.id}`)
-    .then(({ json: { data } }) => data);
+    .then(({ json: { data } }) => data)
+    // eslint-disable-next-line handle-callback-err
+    .catch(err => {});
```

Why is it ok to swallow the error here? Could you add a comment please?